terminal/terminal.md

```diff
@@ -0,0 +1,417 @@
+# The amazing world of the Terminal
+
+The terminal is a command-line interface app to control the operating system with text based commands
+
+To understand what the terminal is we need to ask ourselves, if the user interface in the computer would not exist how we would be able to operate it normally as we do? move across the directories and editing files? 
```

Comment:
`if the user interface in the computer did not exist how would we be able to operate it normally...`